packages/react-vapor/src/components/browserPreview/tests/BrowserPreviewEmpty.spec.tsx

```diff
@@ -39,12 +39,19 @@ describe('BrowserPreviewEmpty', () => {
         expect(defaultProps.onClick).toHaveBeenCalledTimes(1);
     });
 
-    it('renders a Svg as child', () => {
+    it('renders a Svg as child by default', () => {
         mountWithProps(defaultProps);
 
         expect(component.find('.browser-preview__state').childAt(0).find(Svg)).toBeTruthy();
     });
 
+    it('renders the specified node instead of the default image', () => {
+        const image = <img src="fake" />;
+        mountWithProps({...defaultProps, image});
+
+        expect(component.find('.browser-preview__state').childAt(0).find('img[src*="fake"]')).toBeTruthy();
```

Comment:
I would make the test a bit more resilient to change:
```suggestion
        expect(component.find('img[src*="fake"]').exists()).toBe(true);
```